portfolio/src/main/webapp/gallery.html

```diff
@@ -19,11 +19,115 @@
         </nav>
       </div>
 
-      <h1 id="pagename">Himani's Image Gallery</h1>
-
-      <div class="images">
-          <!-- TODO: Add images here -->
+      <div class="gallery-area">
+        <h1 id="pagename">Himani's Image Gallery</h1>
+        <p>I love to travel! Click on an image below to see a description.</p>
+        <div class="image-container">
+          <div class="image" onclick="showCaption(1)">
+            <img src="/images/gallery/quad.jpg" alt="Myers Quad">
+            <span class="caption" id="myCaption1">This is Myers Quad, my favorite part of campus. 
+                I found this gorgeous sunset view outside Myers Hall (my dorm for my first two years at UGA).
+            </span>
+          </div>
+          <div class="image" onclick="showCaption(2)">
+            <img src="/images/gallery/ga-mountains.jpg" alt="Yonah Mountain in Georgia">
+            <span class="caption" id="myCaption2">The gentle scenery of Blue Ridge Mountains in north Georgia is 
+                so pretty. It's a five mile hike up Mt. Yonah, but this view was worth it.
+            </span>
+          </div>
+          <div class="image" onclick="showCaption(3)">
```

Comment:
Since you made it a clickable interaction I have to jump here with a11y (accessibility) tips! Note that none of these changes are required, and feel free to ask questions about any of them.

Switching to a `<button></button>` from a `<div></div>` helps screenreaders know this is interactive.

`<button tabindex=0></button>` will make it so you can tab to the clickable object.

`<button aria-label="Read about Outside UGA Sci Li"></button>` would allow users to tab through all the images and see what they wanted to read more about.

In the same vein, if you wanted to get really fancy when you show the caption in the JS, you could call 

`captionElement.focus()` so that when a user clicks on this button, a screenreader would read your caption aloud.

To get VERY nitpicky, this would be wrong half time time, as the second time a user clicks on this button it wouldn't be showing the caption, but hiding the caption. You could do something super extra like toggle between:

`<button alt="Show caption"></button>`
`<button alt="Hide caption"></button>`

Reply:
I changed the divs to buttons. Now I'm having an issue where images with a horizontal aspect ratio aren't being cropped to fill the whole area like before (I used object-fit: cover; but it doesn't seem to be working anymore). Do you happen to know why this would be happening?

![Screenshot 2020-05-28 at 5 15 08 PM - Display 2](https://user-images.githubusercontent.com/30757425/83194595-e14cbd80-a106-11ea-84b0-72e3333beb68.png)

Reply:
Okay, I couldn't figure out a satisfactory solution for this yet. For now, I've changed the buttons back to divs so that the spacing remains even like it was before. I am definitely interested in making the page more accessible, so hopefully tomorrow I'll have time to make these changes.

Reply:
Button would not be an urgent one. You can actually get most of the same behavior with

```
<div aria-role="button" tabindex=0 aria-label="Show caption"></div>
```

Reply:
tabindex="0" whoops**

Reply:
Okay, great! I'll try that out!